src/main/java/com/google/googleidentity/oauth2/util/OAuth2Constants.java

```diff
@@ -0,0 +1,35 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package com.google.googleidentity.oauth2.util;
+
+/**
+ *  Constants in OAuth2 Request
+ */
+public final class OAuth2Constants {
```

Comment:
nit: `OAuth2ParameterNames`? "Constants" doesn't tell the reader exactly what these strings are. Alternatively, add something like PARAM to each constant name